PrivateRepo.md

```diff
@@ -0,0 +1,159 @@
+### How to Use a Private Repository
```

Comment:
Hmm I'm confused what the point of this is. The template docs describe the method you describe below, or using copier. But regardless, since Survey already is fully initialized with that template code, what is this part adding?

Reply:
It is more about restructuring files. Adding missing tools and removing unnecessary info. It is not easy for me to simply it. Because I have not developed backend in this project & waiting on instructions how to run backend too. Although, about the Front it is clear for me.  So, probably, we have to cooperate with Mitchel, when he will come up with instructions for the Backend...

Reply:
Thanks Daryna.  Yeah. I just want to make it easy to onboard to the project.  To make this easy, it would be great to have a short, clear explanation of all the steps required to set up the run and develop on the front end and back end, maybe with a section at the end on how to troubleshoot if any difficulties occur.

Reply:
Please ask questions on slack if you are having trouble setting up the backend.